src/components/RadioButtonGroup/readme.md

```diff
@@ -372,7 +372,15 @@ const ConctactContainer = styled.div`
     width: max-content;
 `;
 
+<<<<<<< HEAD
+const linkedinIconStyle = {
+    width: "20px",
+    height: "20px",
+    color: "#0E76A8",
+};
+=======
 const linkedinIconStyle = {color: '#0E76A8'};
+>>>>>>> 6ce373d340621fba600f33896ca1800d599409a5
```

Comment:
fix conflicts since this actually looks broken:
<img width="935" alt="Screen Shot 2020-03-10 at 12 28 01 PM" src="https://user-images.githubusercontent.com/24737097/76346414-96bb6200-62ca-11ea-9c26-e77d80ea2660.png">